### DISCUSSION TITLE
1.4.1-beta3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#292] [Обновления]
## Список изменений
- Исправлена ошибка из-за которой язык видео мог определяться в формате отличном от "xx"
- Улучшена логика работы обновлений indexedDB
- Возвращен пункт меню "Проксировать полученное аудио"

## What's Changed (autogenerated)
* README.md: Add support iOS for UserScripts by @ACherepkov1989 in https://github.com/ilyhalight/voice-over-translation/pull/280
* Some fixes by @MrSoczekXD in https://github.com/ilyhalight/voice-over-translation/pull/288

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.4.1-beta2...1.4.1-beta3

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.4.1-beta3'>1.4.1-beta3</a>.</em>

**Comment by NullVerdict** [comment]
Есть очень старый баг, если языка нет в списке, он всё ровно начнет перевод.
Например тут nl в субтитрах. А Яндекс его не поддерживает. Предлагаю переводить на Английский или уточнять язык у сервер.
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/a05bc29d-4bbc-4c78-8ab9-cbb61463c516)

**Reply by ilyhalight** [reply]
Можешь скинуть видео где такое появляется

**Reply by NullVerdict** [reply]
> Можешь скинуть видео где такое появляется

[Вот](https://www.youtube.com/watch?v=ZZ5LpwO-An4)

**Reply by NullVerdict** [reply]
Черные субтитры на мобильной версии

![изображение](https://github.com/ilyhalight/voice-over-translation/assets/24498484/bc93c3c3-119e-4825-a158-7b9494ac321a)

**Reply by ilyhalight** [reply]
> Черные субтитры на мобильной версии

Проверил через фейк мобильную версию в Firefox Dev. Edition 117, там, всё нормально с цветом субтитров, даже, с включенным DarkReader.
Проверил через Kiwi Browser 116 и Firefox Nightly 118, везде, использовал Tampermonkey. И там, и там субтитры были нормальными. 

Скорее всего, у тебя что-то в системе/браузере включено, если ты это тестировал именно на телефоне.

**Comment by NullVerdict** [comment]
При переходе на другое видео с включенным переводом, звук на видео останется 15% а не вернется к старому значению.
